docs/reference/readme.md

```diff
@@ -18,11 +18,11 @@ Pomerium can be configured using a configuration file ([YAML]/[JSON]/[TOML]) or
 
 Using both [environmental variables] and config file keys is allowed and encouraged (for instance, secret keys are probably best set as environmental variables). However, if duplicate configuration keys are found, environment variables take precedence.
 
-Pomerium will automatically reload the configuration file if it is changed. At this time, only policy is re-configured when this reload occurs, but additional options may be added in the future. It is suggested that your policy is stored in a configuration file so that you can take advantage of this feature.
+Pomerium will automatically reload the configuration file if it is changed.
```

Comment:
This is more true than it used to be but not 100% coverage.  We might want to mark what is static (I believe it is only a subset of options).

Reply:
How about now?